src/native/corehost/browserhost/CMakeLists.txt

```diff
@@ -107,7 +107,6 @@ target_link_options(browserhost PRIVATE
     -sINITIAL_MEMORY=134217728
     -sMAXIMUM_MEMORY=2147483648
     -sALLOW_MEMORY_GROWTH=1
-    -sALLOW_TABLE_GROWTH=1
```

Comment:
why we are removing this ? If we ever JIT it would be necessary.

Reply:
we can always add it later if we add jit. the default is 0 and there might be perf or size consequences